src/components/LocationDetails.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/LocationDetails.css";
+
+function LocationDetails(props) {
+  const { city, country, errorMessage } = props;
```

Comment:
You can destructure in the function definition like this:
`function LocationDetails({ city, country, errorMessage }) {`